docs/DesignTokens/Taxonomy.stories.mdx

```diff
@@ -0,0 +1,138 @@
+import { Meta } from "@storybook/addon-docs";
+
+<Meta title="Design Tokens Taxonomy" />
+
+# 🚧 WIP
+
+## Design system namespace
+
+All tokens start with the namespace for the design system, `cpd`.
+
+## Token tiers
+
+Our design tokens are conceptually divided into three tiers:
+
+- System
+- Reference
+- Components
+
+The top level is global and not aware of context. Subsequent tiers are context-aware, each one being more precise.
+
+System tokens describe plain values, while reference tokens describe their purpose in specific contexts. Component tokens describe the intended use in the context of a specific component.
+
+With three kinds of tokens, we can update design decisions globally or apply a change to a single component.
+
+### System
+
+Namespace: `sys`
+🔴 TODO: Consider removing namespace.
+
+Use for global, immutable literal values.
```

Comment:
Nothing currently lets us define the namespace from the token structure

Reply:
After having thought more about this lately I think we can get rid of the token tier namespace, to make things easier to consume / more usable. As we discussed, we could define some linting rules somehow where necessary.

Reply:
That said, I still suggest keeping them as a conceptual thing, but without introducing a namespace (for usability reasons). This way the conceptual distinction will be clear for everyone, and we'll be able to point out people to a resource that details it.

Reply:
Then in that case should we remove all references to the namespace but keep the different tiers as concepts?

Reply:
Completely reworked the documentation on token tiers. 👍